SBDL.h

```diff
@@ -385,7 +385,14 @@ namespace SBDL {
 	* @return font which is loaded
 	*/
 	Font *loadFont(const std::string &path, int size) {
-		return TTF_OpenFont(path.c_str(), size);
+		Font *font;
+		font = TTF_OpenFont(path.c_str(), size);
+		if (!font) {
```

Comment:
font == nullptr
I think using nullptr check is better. Can you please test this and do this for others like loadSound,...